<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SdNdeDbViewWidget</class>
 <widget class="QWidget" name="C_SdNdeDbViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="pc_VerticalLayout" stretch="0,0,0,1">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="pc_VerticalLayout2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="pc_HorizontalLayout" stretch="0,1,0,0">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="stw_opensyde_gui_elements::C_OgeLabHeadingGroupBold" name="pc_LabelApplicationName">
           <property name="text">
            <string>Data Blocks</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="pc_HorizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="stw_opensyde_gui_elements::C_OgePubSvgIconOnly" name="pc_PushButtonCodeGenerationOptions">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="stw_opensyde_gui_elements::C_OgePubIconOnly" name="pc_PushButtonAdd">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="pc_VerticalSpacer2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>4</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="stw_opensyde_gui_elements::C_OgeFrameSeparator" name="pc_FrameApplicationsViewSperator">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="pc_VerticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="stw_opensyde_gui_elements::C_OgeLabGroupItem" name="pc_LabelNoDatablocks">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>No Data Blocks declared...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="stw_opensyde_gui::C_SdNdeDbListWidget" name="pc_ListWidget"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabGroupItem</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGroupItem.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabHeadingGroupBold</class>
   <extends>QLabel</extends>
   <header>C_OgeLabHeadingGroupBold.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeFrameSeparator</class>
   <extends>QFrame</extends>
   <header>C_OgeFrameSeparator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgePubIconOnly</class>
   <extends>QPushButton</extends>
   <header>C_OgePubIconOnly.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgePubSvgIconOnly</class>
   <extends>QPushButton</extends>
   <header>C_OgePubSvgIconOnly.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui::C_SdNdeDbListWidget</class>
   <extends>QListWidget</extends>
   <header>C_SdNdeDbListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
